recipes/autocfg/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "autocfg" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autocfg-{{ version }}.tar.gz
+  sha256: 749986b4f3b3bd85b15298734bf8fa4a590e6c34a314ac515025e058ed76c319
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyyaml
+    - dataclasses  # [py<37]
```

Comment:
This can be made noarch if we drop py37 or if we install `dataclasses` everywhere. I never experimented with the latter but we do that for importlib-metadata, etc.  Do you want to give it a go and see if it works?

Reply:
I think it should work: https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages

Reply:
I have changed to noarch by dropping `[py<37]`